userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/Constants.java

```diff
@@ -29,6 +29,7 @@
 
 public class Constants {
     enum AuthorityType {
+        AAD_GUEST("https://login.microsoftonline.com/nomfaad.onmicrosoft.com"),
```

Comment:
nit: It could be more convenient to put the AAD_MSDEVEX as default at the top of the list, in the perspective of manual testing. So we do not need to scroll down each time.

Reply:
This is a good point, I'd be happy to see this change

Reply:
fixed